### DISCUSSION TITLE
ValidadeAposVencimento em branco na leitura de cobv/{PixUrlAccessToken}

### DISCUSSION THREAD

**Discussion by mvmarcal** [bacen/pix-api#329] [Q&A] [answered]
Percebi que na criação de uma cobrança com vencimento e também no retorno do cobv/{pixUrlAccessToken} o campo ValidadeAposVencimento não é obrigatório, e tem definido que o valor Default é 30 dias.
Abaixo minhas dúvidas:

1. Como na criação o PSP Recebedor deverá garantir que ao menos o Valor Default(30 dias) será preenchido, por que este campo é opcional para retorno do cobv/{PixUrlAccessToken}?
2. Como PSP Pagador, ao consultar este endPoint, caso o valor não seja retornado, devo então assumir que é de 30 dias?

**Comment by ninrod** [comment, marked as answer]
> Como na criação o PSP Recebedor deverá garantir que ao menos o Valor Default(30 dias) será preenchido, por que este campo é opcional para retorno do cobv/{PixUrlAccessToken}?

C> omo PSP Pagador, ao consultar este endPoint, caso o valor não seja retornado, devo então assumir que é de 30 dias?


Ele é opcional na entrada, não no retorno. É um bug na API que você acabou de encontrar. Pode converter em issue por favor que a gente resolve na 2.2.2

**Comment by mvmarcal** [comment]
Criada a issue https://github.com/bacen/pix-api/issues/331 para tratar este item.
